src/packages/@ncigdc/components/Modals/FirstTimeModal.js

```diff
@@ -20,17 +20,18 @@ const FirstTimeModal = () => (
     </p>
     <p>
       <b>WARNING</b>
-      : Please read the
+      : Data in the GDC is considered provisional as the GDC applies state-of-the art analysis pipelines which evolve over time. Please read the
       {" "}
+
       <ExternalLink
         hasExternalIcon={false}
         href="https://docs.gdc.cancer.gov/Data/Release_Notes/Data_Release_Notes"
       >
         GDC Data Release Notes
-      </ExternalLink>
-      {" "}
-      prior to accessing this web site as the Release Notes provide details about data updates, known issues and workarounds.
+      </ExternalLink>&nbsp;
```

Comment:
`&nbsp;` is different though isn't it? the line won't wrap around it. If it is just used to get around prettier I don't think we should use it.